python/README.md

```diff
@@ -29,7 +29,7 @@ part of the default Cargo workspace so that it doesn't cause overhead for mainta
 Creates a new context and connects to a Ballista scheduler process.
 
 ```python
-from pyballista import SessionContext
+from ballista import RemoteBallista, StandaloneBallista
```

Comment:
this readme does not reflect `Ballista` builder pattern

Reply:
fixed